commit faef7e9214e865e369c780bd6182ca306560b1aa
Author: Sylvestre Ledru <sylvestre@debian.org>
Date:   Sat Mar 27 10:05:13 2021 +0100

    fix(install): Unbreak the CI by bringing the old behavior for install of /dev/null

Index: coreutils/src/uu/install/src/install.rs
===================================================================
--- coreutils.orig/src/uu/install/src/install.rs
+++ coreutils/src/uu/install/src/install.rs
@@ -494,10 +494,6 @@ fn copy_file_to_file(file: &PathBuf, tar
 /// If the copy system call fails, we print a verbose error and return an empty error value.
 ///
 fn copy(from: &PathBuf, to: &PathBuf, b: &Behavior) -> Result<(), ()> {
-    if b.compare && !need_copy(from, to, b) {
-        return Ok(());
-    }
-
     if from.to_string_lossy() == "/dev/null" {
         /* workaround a limitation of fs::copy
          * https://github.com/rust-lang/rust/issues/79390
@@ -511,7 +507,13 @@ fn copy(from: &PathBuf, to: &PathBuf, b:
             );
             return Err(());
         }
-    } else if let Err(err) = fs::copy(from, to) {
+    }
+
+    if b.compare && !need_copy(from, to, b) {
+        return Ok(());
+    }
+
+    if let Err(err) = fs::copy(from, to) {
         show_error!(
             "cannot install '{}' to '{}': {}",
             from.display(),
